app/domain/Capabilities.scala

```diff
@@ -52,6 +52,10 @@ object Capabilities {
 
   case object SupportsSubscriptions extends StandardAppCapability with LikePermission
 
+  case object SupportsSubscriptionFields extends Capability {
```

Comment:
Is this for future planning ?  Where it may not always be true ?

Reply:
It was kinda odd that it didn't do this already. Effectively it didn't show the link for ROPC/Priv apps - but you could browse to the URL, which it should have blocked. 

Now it aligned with our capabilities. When we start restricting / blocking access it should easy. It's also a similar pattern we use for capabilities that all apps have (e.g. TeamMembers).

We did question whether it adds a lot when we did it...